extensions/wikia/UserLogin/UserLoginSpecialController.class.php

```diff
@@ -193,7 +193,7 @@ public function getCloseAccountRedirectUrl() {
 	}
 
 	public function modal() {
-		$this->redirectToMercury();
+		$this->redirectToAuthPages();
```

Comment:
We could move this call to `init` method which is automatically invoked by Nirvana dispatcher before each method, so we don't have to call it everywhere.